### PR TITLE
NEXUS-25047 R format. Administrative

### PR DESCRIPTION
**Pull request by mlukaretkyi**
https://issues.sonatype.org/browse/NEXUS-25047

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,13 @@
...
 [![Maven Central](https://img.shields.io/maven-central/v/org.sonatype.nexus.plugins/nexus-repository-r.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.sonatype.nexus.plugins%22%20AND%20a:%22nexus-repository-r%22)
 
+> **Huzzah!** R is now part of Nexus Repository Manager. Version 3.next includes the R plugin by default. The plugin source code is now in [nexus-public](https://github.com/sonatype/nexus-public) in [nexus-repository-r](https://github.com/sonatype/nexus-public/tree/master/plugins/nexus-repository-r)
+
+> **Filing issues:** If using 3.next or later please file any issues at https://issues.sonatype.org/.
```
Comment (jlstephens89) [inline README.md:22]: This implies that if you are using an older version of this plugin that you should file an issue somewhere else.

How about

```
Filing issues: Upgrade to the latest version of Nexus Repository Manager 3, to get the latest fixes and improvements, before filing any issues or feature requests at https://issues.sonatype.org/.
```

File: README.md [hunk 2]
```diff
@@ -17,6 +17,13 @@
...
+
+> **Upgrading to 3.next:** If you are an existing R user upgrading to 3.next you will not be able to install the
```
Comment (jlstephens89) [inline README.md:24]: We should probably state the 'to' version. i.e. if you using a version prior to 3.20 and upgrade to a newer version you will not be able to install the community plugin. No other changes are required, and your existing data will remain intact.

File: README.md [hunk 3]
```diff
@@ -17,6 +17,13 @@
 [![CircleCI](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-r.svg?style=shield)](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-r) [![Join the chat at https://gitter.im/sonatype/nexus-developers](https://badges.gitter.im/sonatype/nexus-developers.svg)](https://gitter.im/sonatype/nexus-developers?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-r/depshield.svg)](https://depshield.github.io)
 [![Maven Central](https://img.shields.io/maven-central/v/org.sonatype.nexus.plugins/nexus-repository-r.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.sonatype.nexus.plugins%22%20AND%20a:%22nexus-repository-r%22)
 
+> **Huzzah!** R is now part of Nexus Repository Manager. Version 3.next includes the R plugin by default. The plugin source code is now in [nexus-public](https://github.com/sonatype/nexus-public) in [nexus-repository-r](https://github.com/sonatype/nexus-public/tree/master/plugins/nexus-repository-r)
```
Comment (jlstephens89) [inline README.md:20]: 3.next should now be 3.20.

File: README.md (hunk 2 above)
Comment (mlukaretkyi) [reply README.md:24]: @jlstephens89 
let's imagine, code base was moved 3.29 for example. 
Maybe we should say
```
If you using a version prior to 3.20 - 3.28 and upgrade to a newer version you will not be able to install the community plugin. No other changes are required, and your existing data will remain intact.
```
I don't know version, and that's why I put 3.next

Comment (jlstephens89) [reply README.md:24]: This is true for all versions 3.20 and later. The range would just be confusing. 

Does it matter where the source lived for 3.20 to 3.28? As long as we're clear it is in a different place now I'm happy.

File: README.md (hunk 1 above)
Comment (joedragons) [reply README.md:22]: If we're not going to disable issues here, you may also say don't use issues here :)

File: README.md (hunk 2 above)
Comment (mlukaretkyi) [reply README.md:24]: I thought about it and it doesn't matter where the source code lived

File: README.md (hunk 1 above)
Comment (mlukaretkyi) [reply README.md]: https://github.com/sonatype-nexus-community/nexus-repository-r/pull/139/commits/15e957ddc6f9601b900eb54b344bde389f98792e

File: README.md (hunk 2 above)
Comment (mlukaretkyi) [reply README.md]: https://github.com/sonatype-nexus-community/nexus-repository-r/pull/139/commits/15e957ddc6f9601b900eb54b344bde389f98792e

File: README.md (hunk 3 above)
Comment (mlukaretkyi) [reply README.md]: https://github.com/sonatype-nexus-community/nexus-repository-r/pull/139/commits/15e957ddc6f9601b900eb54b344bde389f98792e

File: README.md (hunk 1 above)
Comment (jlstephens89) [reply README.md]: @joedragons we're going to archive this repo which disables issues

**Review by j-s-3 — APPROVED** [review]
👍

**Review by joedragons — APPROVED** [review]

**Review by bhamail — APPROVED** [review]

**Pull request merged**